styles/file-patch-list-view.less

```diff
@@ -30,6 +30,7 @@
 
   &-path {
     flex: 1;
+    word-break: break-all;
```

Comment:
Great suggestion. 👍  Although I wonder if we should use `word-break: break-word;` instead?

It might be easier to read whole words? And a file extension like `.scss` wouldn't break into `.s` and `css`. If a word is too long, it will still start breaking every character like for folder names.

`break-all` | `break-word`
--- | ---
![break-all](https://cloud.githubusercontent.com/assets/378023/26185210/72c26a4a-3bc4-11e7-98bc-b203b080686d.gif) | ![break-word](https://cloud.githubusercontent.com/assets/378023/26185211/72cd66ac-3bc4-11e7-952e-5bded2796de4.gif)